### DISCUSSION TITLE
Where can I get the dataset that include titiles and abstract?

### DISCUSSION THREAD

**Discussion by JackWrl** [asreview/asreview#1313] [Q&A] [answered]
After set up the new project in ASReview. It wants me to Add dataset, where can I get the suitable dataset?

**Comment by rohitgarud** [comment, marked as answer]
Hi @JackWrl .. if you just want example datasets to get familiar with ASReview or want to test some new algorithm you can use datasets from [Systematic Review Datasets](https://github.com/asreview/systematic-review-datasets) repository

**Reply by Rensvandeschoot** [reply]
Additionally, you can find more explanation of the active learning cycle and the role of data in our [blog post](https://asreview.nl/blog/active-learning-explained/).

**Reply by JackWrl** [reply]
> Hi @JackWrl .. if you just want example datasets to get familiar with ASReview or want to test some new algorithm you can use datasets from [Systematic Review Datasets](https://github.com/asreview/systematic-review-datasets) repository

So, If I want to search for papers in my related field, I have to use a web crawler (or other technique) to save at least titles and abstracts from the webpage into the dataset, and then upload them into ASReview for further active learning?

**Reply by rohitgarud** [reply]
@JackWrl If you have access to databases like [Scopus](https://www.scopus.com/sources.uri?zone=TopNavBar&origin=) or Web of Science through your university, you can search articles relevant to your field using keywords and export the records in .csv or .ris file, which can be imported into ASReview. I suggest first watching a few videos about how systematic literature review(SLR) is performed, what is PRISMA framework, etc. to get yourself familiarised with the SLR process, then you can better understand why and how ASReview is helpful in the SLR process. Also learning about reference managers like Endnote (paid), Zotero (free), and Mendeley (free) will be helpful for you. Hope this helps.

**Reply by JackWrl** [reply]
> @JackWrl If you have access to databases like [Scopus](https://www.scopus.com/sources.uri?zone=TopNavBar&origin=) or Web of Science through your university, you can search articles relevant to your field using keywords and export the records in .csv or .ris file, which can be imported into ASReview. I suggest first watching a few videos about how systematic literature review(SLR) is performed, what is PRISMA framework, etc. to get yourself familiarised with the SLR process, then you can better understand why and how ASReview is helpful in the SLR process. Also learning about reference managers like Endnote (paid), Zotero (free), and Mendeley (free) will be helpful for you. Hope this helps.

Thank you for the detailed reply, now I clearly know what to do. 
Scopus has login problems outside the university internet and the download button for the list does not work, so I was confused about the dataset. I thought maybe I need to collect all information about the list of papers manually (such as web crawler) from the journal webpage. Hope the network in the university will work for Scopus.

**Comment by FelixWdm** [comment]
@JackWrl for a free database, check https://www.lens.org/lens/search/scholar/structured. where you can easily export RIS files. Still Endnote/Zotero would be good to use to collect your search results.
